docs/src/pages/components/box/box.md

```diff
@@ -34,15 +34,18 @@ This works great when the changes can be isolated to a new DOM element.
 For instance, you can change the margin this way.
 
 However, sometimes you have to target the underlying DOM element.
-For instance, you want to change the border of the Button.
+As an example, you may want to change the border of the Button.
 The Button component defines its own styles. CSS inheritance doesn't help.
 To workaround the problem, you have two options:
 
-1. Use [`React.cloneElement()`](https://reactjs.org/docs/react-api.html#cloneelement)
+1. Use the [`sx`](/system/basics/#the-sx-prop) prop directly on the child if it is Material-UI component
 
-The Box component has a `clone` prop to enable the use of the clone element method of React.
-
-{{"demo": "pages/components/box/BoxClone.js", "defaultCodeOpen": true }}
+```diff
+-<Box sx={{ border: '1px dashed grey' }}>
+-  <Button>Save</Button>
+-</Box>
++<Button sx={{ border: '1px dashed grey' }}>Save</Button>
+```
 
 2. Use render props
```

Comment:
It looks like we can drop this as well. It was added to solve the same problem that `sx` handles now.

Maybe a follow up or in the same PR? I mean, the problem we go after is the same, so happy with both.

Reply:
A follow up PR is better for the release notes.